app/models/nomination_email.rb

```diff
@@ -2,14 +2,41 @@
 
 class NominationEmail < ApplicationRecord
   belongs_to :school
+  belongs_to :partnership_notification_email, optional: true
```

Comment:
Can we add the association boilerplate spec for this as well, please.

Reply:
And i haven't quite understood why we need this association.

Reply:
@amazimbe I added this association for analytics purposes. We want to be able to link a clicked nominate link back to the partnership email, and a partnership email notify status through to a nomination email record

Reply:
Since some of the partnership notification emails generate a nomination email record without actually sending a separate nomination email (We need the token for a nomination)